lib/utils/Utils.js

```diff
@@ -37,7 +37,7 @@ const Utils = {
   },
 
   clog: function(where, msg, obj) {
-    if (process.env.SERVER_ENV || this.logLevel < LOG_LEVEL_INFO) {
+    if (this.logLevel < LOG_LEVEL_INFO) {
```

Comment:
I think this should be `process.env.NODE_ENV === 'production'` or something.

We don't want the log outputs on production. It's a mess. Will lower the response time heavily.